### DISCUSSION TITLE
Saved directories

### DISCUSSION THREAD

**Discussion by Ray7301** [AlUlkesh/stable-diffusion-webui-images-browser#249] [Q&A] [answered]
Question, I use custom directories for specific renders, i move the pictures to those folders, but i have to copy paste the folder information into the directory path everytime and adjust it, now below it is saved directories. It is my hope that i can place my directories in there and select them from the pull down menu, but i have no clue how to fill that pull down menu. (its when you open a picture in the Image browser and select the image. (could also be that i am asuming its that but its is for something else entirely. I can't find any information about it (might be looking in the wrong location. Anyone any idea? <img width="633" alt="image" src="https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/149721357/af33bf1a-6892-4b34-b978-8a98edf87006">

**Comment by AlUlkesh** [comment, marked as answer]
That "Saved directories" field is an extension to the "Others" tab. So you can save save your folders there and they should be available on the other tabs. Does this answer your question?

![image](https://github.com/AlUlkesh/stable-diffusion-webui-images-browser/assets/99896447/752d8782-f833-4822-ac6e-7b7e1a6da460)

**Reply by Ray7301** [reply]
Yes it does, so sorry when adding my custom directories i removed Others, figured i didn't need it at the time so i never saw the saved directories there. So when i was searching i never connected the two. (doh). I love your extention btw, would not be using Automatic 1111 without it.

**Reply by AlUlkesh** [reply]
Well, that certainly explains it. Glad you got it figured out.

Thanks 😄
